platform/android/display_server_android.h

```diff
@@ -39,6 +39,9 @@ class RenderingDeviceVulkan;
 #endif
 
 class DisplayServerAndroid : public DisplayServer {
+	//No need to register, it's platform-specific and nothing is added
+	//GDCLASS(DisplayServerAndroid, DisplayServer)
```

Comment:
```suggestion
	// No need to register with GDCLASS, it's platform-specific and nothing is added.
```

Reply:
Same for the others.

Reply:
You missed some changes from my suggestion:
- Space after `//`
- Sentence ends with a period

Reply:
Fixed 😅